software/bsw/mcal/dio_swc/tools/cmake/dio_swc_tests.cmake

```diff
@@ -2,31 +2,54 @@ set(
     dio_swc_tests_sources_list
 
     ${dio_swc}/tests/test_dio_swc.c
-    ${dio_swc}/tests/stubs/test_dio_swc_stubs.c
+
+    # Any mocked file will be under stub/mock_headerFileName.c
+    ${dio_swc}/tests/stubs/mock_Det.c
+    # Include Test Runners 
+    ${dio_swc}/tests/test_runners/dio_Runner.c
+    ${dio_swc}/tests/test_runners/all_tests.c
 )
 
 set(
     dio_swc_ut_sources_list
-
+    
```

Comment:
to decrease the CMake dependency fingerprint we can export one variable with common sources and headers and include it like this 

add_executable(dio_swc_test ${dio_swc_tests_sources_list} ${unity_common_sources})